### ISSUE TITLE
Update peer Dependencies to support Angular 7 without warnings

### ISSUE THREAD

**Issue by DanielHabenicht** [ike18t/ng-mocks#23]
Currently when installed with Angular 7 npm produces three warnings:
```bash
npm WARN ng-mocks@6.2.1 requires a peer of @angular/compiler@>=5.x <=6.x but none is installed. You must install peer dependencies yourself.
npm WARN ng-mocks@6.2.1 requires a peer of @angular/core@>=5.x <=6.x but none is installed. You must install peer dependencies yourself.
npm WARN ng-mocks@6.2.1 requires a peer of @angular/forms@>=5.x <=6.x but none is installed. You must install peer dependencies yourself.
```

I did not encounter any problems when using the current version (6.2.1) with Angular 7

**Comment by DanielHabenicht** [comment]
Nevermind, my packages were still outdated...

**Comment by ike18t** [comment]
Fixed with release 7.0.0
